web/default_preferences.json

```diff
@@ -0,0 +1,20 @@
+{
+  "showPreviousViewOnLoad": true,
+  "defaultZoomValue": "",
+  "sidebarViewOnLoad": 0,
+  "cursorToolOnLoad": 0,
+  "enableWebGL": false,
+  "pdfBugEnabled": false,
+  "disableRange": false,
+  "disableStream": false,
+  "disableAutoFetch": false,
+  "disableFontFace": false,
+  "textLayerMode": 1,
+  "useOnlyCssZoom": false,
+  "externalLinkTarget": 0,
+  "renderer": "canvas",
+  "renderInteractiveForms": false,
+  "enablePrintAutoRotate": false,
+  "disablePageMode": false,
+  "disablePageLabels": false
+}
```

Comment:
Not sure what to do with this yet. I don't like where it is but pdfjs looks for it in a very particular place.